lms/static/sass/course/_gradebook.scss

```diff
@@ -75,7 +75,9 @@ div.gradebook-wrapper {
 		position: relative;
 		float: left;
 		width: 76%;
-		overflow: hidden;
+		overflow: auto;
```

Comment:
Which browsers don't support `overflow-x` and `overflow-y`? Are they on [the list of browsers that Open edX supports](http://edx.readthedocs.org/projects/edx-installing-configuring-and-running/en/named-release-cypress/front_matter/browsers.html)?